### PR TITLE
[[ Documentation ]] Fixes to call.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
- Added links to referenced tokens.
- Added messages tag.
- Added object reference as reference item.
- Disambiguate handler and object params in syntax.

### CODE REVIEW

File: docs/dictionary/command/call.lcdoc [hunk 1]
```diff
@@ -21,46 +21,49 @@ Example:
...
 Description:
 Use the <call> <command> to use a <handler> that's not in the normal 
 <message path>.
 
 The <call> <command> sends a <handler> <message> to the 
-<object(glossary)>. If the <script> of the <object(glossary)> doesn't 
+<object>. If the <script> of the <object> doesn't 
 <trap> the <handler> <message>, the <message> is <pass|passed> to the 
-next <object(glossary)> in the object's <message path>.
+next <object> in the <object|object's> <message path>.
```
Comment (livecodeali) [inline docs/dictionary/command/call.lcdoc:41]: Unfortunately `<object(glossary)>` needs to stay like that because `<object>` is a valid html tag

**Comment by asayd** [comment]
@livecodeali Ah, so it does. Must add that to my checklist.

**Comment by montegoulding** [comment]
👍 
@livecode-vulcan review ok 68c0a28

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 68c0a28445eae748e4ce2d11b784bc1d1696d829

**Comment by livecode-vulcan** [comment]
😎 test success 68c0a28445eae748e4ce2d11b784bc1d1696d829
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/97)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/97)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/97)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/97)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/97)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/97)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/97)

**Pull request merged**